frontend/src/utilsLogic.tsx

```diff
@@ -201,15 +201,16 @@ export const getManagerInstances = (
   index: number
 ) => {
   const { metadata, spec, status } = instance!;
-  const { environmentList, templateName } = spec
-    ?.templateCrownlabsPolitoItTemplateRef?.templateWrapper
+  const { name, namespace, templateWrapper } =
+    spec?.templateCrownlabsPolitoItTemplateRef as any;
```

Comment:
Is the `as any` required?

Reply:
(also below)

Reply:
unfortunately yes, for the same ever reason